### PR TITLE
added product page

### PR DESCRIPTION
**Pull request by MurtazaBagwala07**
made context for managing data
made reducer function to load data
created product page using diff components

### CODE REVIEW

**Review by hrishi82 — APPROVED** [review]
Liked the modular file structure
Rest of the code looks good to me

File: jersea/src/App.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import "./App.css";
-import {Home} from './pages/Home/Home.jsx'
+import {Home,ProductPage} from './pages/index'
+import { Header } from "./components/index";
```
Comment (anuj-kumary) [inline jersea/src/App.js:3]: No need to write **index** also when you doing barrel import.
This also work 
`import {Home,ProductPage} from './pages'`

File: jersea/src/components/Products/ProductFilter/ProductFilter.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            <span>Filter</span>
+            <span>Clear</span>
+        </div>
+        <div className="ecom-filter-price">
+            <label for="price">Price</label>
+            <input type="range"/>
+        </div>
+        <div className="ecom-filter-category flex-column">
+            <p for="category">Category</p>
+            <label for="league"><input className="filter-label" type="checkbox"/>Laliga</label>
```
Comment (Rohanmond) [inline jersea/src/components/Products/ProductFilter/ProductFilter.jsx:17]: instead of harcoding you can use db categories array

**Review by Rohanmond — APPROVED** [review]

**Review by anuj-kumary — APPROVED** [review]
Liked your folder structure also your code is very clear and readable

**Pull request merged**